scripts/git-hooks/pre-commit

```diff
@@ -13,3 +13,32 @@ do
     goimports -w "${file}"
     git add "$file"
 done
+
+# Mypy doesn't support recursive traversal of directories
+# So we manually call it on every staged python file
+# and OR together the error codes to see if at least one failed
+exit_status=0
```

Comment:
```suggestion
local exit_status=0
```

I think with our bash scripts we should be as vigilant as possible to reduce the scope of our variables (even if it's just a single pre-commit hook!)